docs/usage/configuration-options.md

```diff
@@ -3022,7 +3022,7 @@ We recommend you use only one of these methods, or you'll get confused.
 We recommend that you also tell Renovate what `versioning` to use.
 If the `versioning` field is missing, then Renovate defaults to using `semver` versioning.
 
-For more details and examples, see our [documentation for the `regex` manager](/modules/manager/regex/).
+For more details and examples, see our [documentation for the `regex` manager](/modules/manager/regex/index.md).
```

Comment:
We also have links like this:

`https://docs.renovatebot.com/modules/manager/regex/`

What do you want me to do with those?

Reply:
Is it not possible to have relative links which work in github and on the website?

Reply:
currently not, because the file structure for custom manager don't match the generated docs structure. 😕